hudi-timeline-service/src/main/java/org/apache/hudi/timeline/service/RequestHandler.java

```diff
@@ -135,7 +135,7 @@ private boolean syncIfLocalViewBehind(Context ctx) {
       synchronized (view) {
         if (isLocalViewBehind(ctx)) {
           HoodieTimeline localTimeline = viewManager.getFileSystemView(basePath).getTimeline();
-          LOG.info("Syncing view as client passed last known instant " + lastKnownInstantFromClient
+          LOG.debug("Syncing view as client passed last known instant " + lastKnownInstantFromClient
```

Comment:
this is again rather important. Going to just have the last instant printed out.